src/Fabulous.MauiControls/Views/Shapes/Polyline.fs

```diff
@@ -24,14 +24,14 @@ module Polyline =
             | ValueSome string -> target.SetValue(Polyline.PointsProperty, PointCollectionConverter().ConvertFromInvariantString(string)))
 
     let PointsList =
-        Attributes.defineSimpleScalarWithEquality<Point array> "Polyline_PointsList" (fun _ newValueOpt node ->
+        Attributes.defineSimpleScalarWithEquality<Point seq> "Polyline_PointsList" (fun _ newValueOpt node ->
```

Comment:
It is always better to store `array` than `seq` in attributes as it is more optimized.
If you want people to pass in a `seq`, you'll need to do the conversion in the modifier directly (`PointsList.WithValue(Array.ofSeq value)`)